### ISSUE TITLE
[pol] Dates make the resource look outdated

### ISSUE THREAD

**Issue by yatil** [w3c/wai-planning-and-implementation#38]
> By February 1, 2015, ACME Inc's web content will meet WCAG 2.0, Level AA conformance.
> 
> By November 1, 2014, ACME Inc's websites will meet WCAG 2.0, Level A conformance; and by May 1, 2015 web content will meet WCAG 2.0, Level AA conformance.
> 
> By June 1, 2015, all vendors of content management systems (CMS) used by ACME Inc's should provide information regarding their plans for ATAG 2.0 conformance in future versions of their software. By June 1, 2016 ACME Inc. will preferentially purchase ATAG conformant authoring tools.

I would love to have that dynamic – using a PHP snippet that always outputs the nearest February or November, for example. But for now I’d just suggest to change the dates to 2016/2017…

**Comment by iadawn** [comment]
Discussion in EO plan agreed to change to 20XX to avoid any problems with shelf life

**Comment by shawna-slh** [comment]
I also think that is much better for several reasons -- e.g., people should have had policies that they will conform to WCAG years ago, so I'm not keen on having later years.

**Comment by yatil** [comment]
Thanks, @iadawn!

**Comment by iadawn** [comment]
@slhenry, you get no argument from me on what people **should** have been doing ;)
